### DISCUSSION TITLE
Please remove YOLO or provide an option to hide it

### DISCUSSION THREAD

**Discussion by ronnygunawan** [community/community#18340] [Other Feature Feedback, Questions, & Ideas]
I'm sure people who feel that this badge can affect their chance of getting their dream job will be upset or even create a second account.

**Comment by ale-lit** [comment]
Hi! You can turn it off in the settings: https://github.com/settings/profile#profile-settings-heading

![image](https://user-images.githubusercontent.com/91823697/173179844-783d10ac-cdf8-4232-929c-810c141fd1ea.png)

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile#changing-the-visibility-of-achievements

**Reply by ronnygunawan** [reply]
But unchecking that option will also hide my shark

**Reply by ale-lit** [reply]
Yes, it's either all or nothing unfortunately =(

**Reply by ale-lit** [reply]
I don't think you should worry too much about this icon.

**Comment by melvincwng** [comment]
Hi @ronnygunawan , as of right now, there is no way to remove any specific achievement/badge from your Github Profile page.

Probably would have to wait till Github decides to implement that functionality.

**Comment by Belikeoscar** [comment]
I don't like YOLO either, I feel it gives off the wrong message having that badge , putting into consideration that some people working on friendly projects would easily merge without code reviews and having the badge makes someone seem careless .
